google/cloud/forseti/services/inventory/storage.py

```diff
@@ -154,6 +155,64 @@ def set_error(self, session, message):
         session.add(self)
         session.flush()
 
+    def get_lifecycle_state_details(self, session, resource_type_input):
+        """Generate/return the count of lifecycle states (ACTIVE, DELETE_PENDING) of the specific
```

Comment:
The summary sentence for the method doc should fit in one line, and move the detailed descriptions to a follow-up paragraph.  So, you can say:
```
"""Count of lifecycle states of the specific resources.

Generate/return the count of lifecycle states (ACTIVE, DELETE_PENDING) of the specific
resource type input (project, folder) for this inventory index.
"""

Reply:
Done and done.